assets/react/controllers/Delivery/DeliveryForm.jsx

```diff
@@ -170,12 +174,13 @@ export default function ({ storeId, deliveryId, order }) {
       }
     }
 
-    return Object.keys(errors.tasks).length > 0 ? errors : {}
-  }
 
-  // Could not figure out why, but sometimes Formik "re-renders" even if the values are the same.
-  // so i store a ref to previous values to avoid re-calculating the price.
-  const previousValues = useRef(initialValues);
+    if (overridePrice && !values.variantName) {
```

Comment:
The behaviour in the old form is to allow an empty product name (as a way to remove the custom name and generate a default name). I think it's good to keep that behaviour at least

![Screenshot 2025-02-10 at 14 46 36](https://github.com/user-attachments/assets/6009ba57-eb09-40ac-8fbd-9477a5cfc83f)

Reply:
ok